pkg/operator/apiservercontrollerset/apiservercontrollerset.go

```diff
@@ -0,0 +1,158 @@
+package apiservercontrollerset
+
+import (
+	"context"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions/config/v1"
+	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/apiservicecontroller"
+	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/nsfinalizercontroller"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/loglevel"
+	"github.com/openshift/library-go/pkg/operator/revisioncontroller"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/unsupportedconfigoverridescontroller"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+	apiregistrationv1client "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1"
+	apiregistrationinformers "k8s.io/kube-aggregator/pkg/client/informers/externalversions"
+)
+
+type APIServerControllerSet struct {
+	operatorClient v1helpers.OperatorClient
+	eventRecorder  events.Recorder
+
+	apiServiceController            *apiservicecontroller.APIServiceController
+	clusterOperatorStatusController *status.StatusSyncer
+	configUpgradableController      *unsupportedconfigoverridescontroller.UnsupportedConfigOverridesController
+	logLevelController              *loglevel.LogLevelController
+	finalizerController             *nsfinalizercontroller.FinalizerController
+	revisionController              *revisioncontroller.RevisionController
```

Comment:
remove

Reply:
we use it for encryption I think

Reply:
removed `RevisionController`, I forgot about it before